schema/fair.js

```diff
@@ -6,67 +6,72 @@ import {
   GraphQLObjectType,
   GraphQLString,
   GraphQLBoolean,
-  GraphQLNonNull
+  GraphQLNonNull,
 } from 'graphql';
 
-let FairType = new GraphQLObjectType({
+const FairType = new GraphQLObjectType({
   name: 'Fair',
   fields: () => ({
-    cached: cached,
+    cached,
     _id: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     id: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     profile: {
       type: Profile.type,
       resolve: ({ default_profile_id, organizer }) => {
-        let id = default_profile_id || organizer && organizer.profile_id;
+        const id = default_profile_id || organizer && organizer.profile_id;
         return gravity(`profile/${id}`);
-      }
+      },
     },
     has_full_feature: {
-      type: GraphQLBoolean
+      type: GraphQLBoolean,
     },
     href: {
       type: GraphQLString,
       resolve: ({ default_profile_id, organizer }) => {
-        let id = default_profile_id || organizer && organizer.profile_id;
+        const id = default_profile_id || organizer && organizer.profile_id;
         return `/${id}`;
-      }
+      },
     },
     start_at: date,
     end_at: date,
     name: {
-      type: GraphQLString
+      type: GraphQLString,
     },
     published: {
-      type: GraphQLBoolean
+      type: GraphQLBoolean,
+      deprecationReason: 'Prefix Boolean returning fields with `is_`',
```

Comment:
I mostly wanted to try this out. Pretty cool; you just give it a deprecationReason and it stops showing up in graphiql, but will still work if you query with it.